draft-ietf-quic-applicability.md

```diff
@@ -318,7 +318,8 @@ HTTP/3 in {{QUIC-HTTP}}. There are a few general principles to apply when
 designing an application's use of streams:
 
 - A single stream provides ordering. If the application requires certain data to
-be received in order, that data should be sent on the same stream.
+be received in order, that data should be sent on the same stream. There is
+no guarantee of transmission ordering across streams.
```

Comment:
would it be more consistent to say that there is no guarantee of reception ordering across streams?

I think the transport caveat for applications using QUIC is that they might write N bytes to stream 0, and N bytes to stream 4 (in order) and expect that the recipient would receive them in that order.

Talking about transmission order is still a useful point but it is slightlly different. A QUIC implementation might allow ordered writes (stream 0 then stream 4, as above) but it might not provide any guarantee of transmission order. Perhaps you can add another sentence to mention transmission with a forward ref to the prioritization section?

Reply:
Yeah. Reception order is better.

```suggestion
be received in order, that data should be sent on the same stream. There is
no guarantee of reception ordering across streams.
```